config/core/test-configs.yaml

```diff
@@ -296,6 +345,33 @@ test_plans:
       job_timeout: '10'
       kselftest_collections: "lkdtm"
 
+  kselftest-membarrier:
+    <<: *kselftest
+    params:
+      job_timeout: '10'
+      kselftest_collections: "membarrier"
+    filters: *kselftest_no_fragment
+
+  kselftest-mincore:
+    <<: *kselftest
+    params:
+      job_timeout: '10'
+      kselftest_collections: "mincore"
+    filters: *kselftest_no_fragment
+
+  kselftest-net:
+    <<: *kselftest
+    params:
+      job_timeout: '10'
+      kselftest_collections: "net"
```

Comment:
I think we might not be able to run this one due to the debug network interface causing issues with kernel-side DHCP.  See this commit https://github.com/kernelci/kernelci-core/commit/ddedee95b11175e9e95941523d0872ce78c0392b